bot/exts/info/information.py

```diff
@@ -518,12 +518,24 @@ async def json(self, ctx: Context, message: Message) -> None:
         await self.send_raw_content(ctx, message, json=True)
 
     @command(aliases=("rule",))
-    async def rules(self, ctx: Context, rules: Greedy[int]) -> None:
-        """Provides a link to all rules or, if specified, displays specific rule(s)."""
+    async def rules(self, ctx: Context, *args: Optional[str]) -> None:
+        """
+        Provides a link to all rules or, if specified, displays specific rules(s).
+
+        It accepts either rule numbers or particular keywords that map to a particular rule.
+        Rule numbers and keywords can be sent in any order.
+        """
         rules_embed = Embed(title="Rules", color=Colour.og_blurple(), url="https://www.pythondiscord.com/pages/rules")
+        keywords, rules = [], []
 
-        if not rules:
-            # Rules were not submitted. Return the default description.
+        for word in args:
+            if word.isdigit():
+                rules.append(int(word))
```

Comment:
`isdigit` is not appropriate because it also considers a bunch of Unicode characters valid, but `int()` will not work with them. The best way is to use `int()` within a try-except catching the `ValueError`.